### DISCUSSION TITLE
Run Regardless of Directory?

### DISCUSSION THREAD

**Discussion by pmo6** [sam-astro/Z-Sharp#37] [Q&A]
How can I make ZSharp run any .zs file, regardless of where it is in? I have tried moving the ZSharp binary to /usr/bin, but it said it failed to change the directory to HelloWorld.zs when I tried to run it. I don't want to have to worry about having potential users of my programs needing the binary for the programming language itself in every single folder a Z# program is located in, so could I please have some assistance here?

EDIT: I was able to get it to work by maintaining the "./" prefix in the ZS file after moving the binary to /usr/bin (and modifying the permissions to run it), but I don't believe any other programming language requires this. Could this please be fixed?

**Comment by sam-astro** [comment]
What is the exact command you are using? Did you add ZSharp to your PATH? Please provide more information.

**Reply by pmo6** [reply]
I guess that would help, but it may not have been made very obvious in the readme files.
